packages/sync/src/utils.ts

```diff
@@ -42,11 +46,12 @@ export function deserializeCrdtDoc(
 		const { document } = JSON.parse( serializedCrdtDoc );
 
 		// Mark this document as from persistence.
-		const docMetaMap = new Map< string, boolean >();
-		docMetaMap.set( CRDT_DOC_META_PERSISTENCE_KEY, true );
+		const docMeta = {
+			[ CRDT_DOC_META_PERSISTENCE_KEY ]: true,
+		};
```

Comment:
Nit: Obviously would throw a type error below, but this would put it closer to the real issue.
```suggestion
		const docMeta: DocumentMeta = {
			[ CRDT_DOC_META_PERSISTENCE_KEY ]: true,
		};
```